odd-platform-api/src/main/java/org/opendatadiscovery/oddplatform/repository/RoleRepository.java

```diff
@@ -4,6 +4,7 @@
 import org.opendatadiscovery.oddplatform.model.tables.pojos.RolePojo;
 
 public interface RoleRepository extends CRUDRepository<RolePojo> {
+
```

Comment:
why?

Reply:
There is no strict checkstyle, with this whitespace looks more nice

Reply:
we have the same style in all files, let's not change it for 1 file